lang/en.js

```diff
@@ -46,13 +46,16 @@ export default {
       "name": "Type of media"
     },
     "LANGUAGE": {
-      "name": "Language"
+      "name": "Language",
+      "plural": "Languages"
```

Comment:
This should use the [vue-i18n pluralization pattern](https://kazupon.github.io/vue-i18n/guide/pluralization.html), if possible.

Reply:
As mentioned above, I don't think NUXT I18N handles pluralization

Reply:
Like this:
```diff
index f7d8406..e017e50 100644
--- a/components/search/MoreFacetsDropdownOptions.vue
+++ b/components/search/MoreFacetsDropdownOptions.vue
@@ -3,7 +3,7 @@
     <strong
       class="mb-4 d-inline-block"
     >
-      {{ $t(`facets.${facet.name}`).name }}
+      {{ $tc(`facets.${facet.name}.name`, 1) }}
     </strong>
     <b-form-checkbox-group
       class="option-group"
@@ -50,10 +50,10 @@
       type="button"
       class="btn btn-link btn-toggle"
       :class="{ 'is-active': isActive }"
-      :data-qa="(isActive ? $t(`facets.button.showLess`, { label: $t(`facets.${facet.name}`).plural }) + ' button' : $t(`facets.button.showAll`, { label: $t(`facets.${facet.name}`).plural }) + ' button')"
+      :data-qa="(isActive ? $t(`facets.button.showLess`, { label: $tc(`facets.${facet.name}.name`, 2) }) + ' button' : $t(`facets.button.showAll`, { label: $tc(`facets.${facet.name}.name`, 2) }) + ' button')"
       @click.prevent="isActive = !isActive"
     >
-      {{ isActive ? $t(`facets.button.showLess`, { label: $t(`facets.${facet.name}`).plural }) : $t(`facets.button.showAll`, { label: $t(`facets.${facet.name}`).plural }) }}
+      {{ isActive ? $t(`facets.button.showLess`, { label: $tc(`facets.${facet.name}.name`, 2) }) : $t(`facets.button.showAll`, { label: $tc(`facets.${facet.name}.name`, 2) }) }}
     </button>
   </b-dropdown-form>
 </template>
diff --git a/lang/en.js b/lang/en.js
index fdb9a2b..11371e7 100644
--- a/lang/en.js
+++ b/lang/en.js
@@ -46,16 +46,13 @@ export default {
       "name": "Type of media"
     },
     "LANGUAGE": {
-      "name": "Language",
-      "plural": "Languages"
+      "name": "Language | Languages"
     },
     "PROVIDER": {
-      "name": "Aggregator",
-      "plural": "Aggregators"
+      "name": "Aggregator | Aggregators"
     },
     "DATA_PROVIDER": {
-      "name": "Institution",
-      "plural": "Institutions"
+      "name": "Institution | Institutions"
     },
     "contentTier": {
       "name": "Includes broken items"
```